mir-datasets.yaml

```diff
@@ -783,6 +864,18 @@ LMD - Latin:
   metadata: 10 genres
   contents: 3160 songs
   audio: 'no'
+  
+LocalifyMusicEvents-USA-2019:
+  url: https://github.com/JimiLab/LocalifyMusicEventData
+  metadata: 
+  contentes: 308051 music events that took place in 2019 and from 1139 US cities
```

Comment:
please fix the type

Reply:
Apologies, I missed this one. I think it is done now.

Reply:
Ah no, it should be `contents` instead of `contentes`

Reply:
<img width="1091" alt="image" src="https://user-images.githubusercontent.com/7326732/208517312-cbc2bda2-872d-48bc-99f8-db53924c9b3a.png">
That's also where the unit test is getting an assertion error.